wasm_bindgen/3rdparty/BUILD.bazel

```diff
@@ -9,35 +9,13 @@ exports_files([
 crates_vendor(
     name = "crates_vendor",
     annotations = {
-        "curl-sys": [crate.annotation(
-            gen_build_script = False,
-        )],
```

Comment:
I was prompted to remove these two annotations when running: `bazel run //wasm_bindgen/3rdparty:crates_vendor -- --repin`:

```
Error: Unused annotations were provided. Please remove them: [CrateId { name: "curl-sys", version: "*" }, CrateId { name: "openssl-sys", version: "*" }]
```